src/components/DatePicker/DatePicker.jsx

```diff
@@ -51,12 +52,10 @@ export const DatePicker = ({
     });
     const [endMonth, setEndMonth] = useState(() => {
         if (!value || !value.to || !value.from) {
-            return dayjs(new Date()).add(1, "month").toDate();
+            return now(new Date()).add(1, "month").toDate();
         }
 
-        return dayjs(value.to).isSame(dayjs(value.from), "month")
-            ? dayjs(value.from).add(1, "month").toDate()
-            : value.to;
+        return now(value.to).isSame(now(value.from), "month") ? now(value.from).add(1, "month").toDate() : value.to;
```

Comment:
Can you add a `isSame` helper. `function whatever(a, b, unit = "somehitng")` So we can do `whatever(value.to, value.from)`

I don't want devs to be using too much dayjs methods